Resources/Textures/Structures/cargo_pallets.rsi/meta.json

```diff
@@ -0,0 +1,17 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+  "copyright": "Created by wafehling (github) partially from a modified creation by EmoGarbage404 (https://github.com/space-wizards/space-station-14/blob/e3180aec030edc6fef25e0262831c2215d9bb2a8/Resources/Textures/Structures/Storage/Crates/generic.rsi/meta.json)",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+             "name": "cargo_pallet_buy"
+        },
+        {
+            "name": "cargo_pallet_sell"
+        }
+    ]
+}
```

Comment:
You should run a formatter over this as it's a bit skewiff.

Reply:
Fixed a couple scraggly spaces, thanks.